src/components/ha-chip.ts

```diff
@@ -9,23 +9,23 @@ import {
   unsafeCSS,
 } from "lit";
 import { customElement, property } from "lit/decorators";
+import { fireEvent } from "../common/dom/fire_event";
 
 declare global {
-  // for fire event
   interface HASSDomEvents {
-    "chip-clicked": { index: string };
+    "chip-clicked": { index: number };
```

Comment:
We can drop this now right ? The code that renders `ha-chip` can just attach a click handler?